packages/vuetify/src/mixins/activatable/index.ts

```diff
@@ -170,7 +170,8 @@ export default baseMixins.extend({
         activator = (e.currentTarget || e.target) as HTMLElement
       }
 
-      this.activatorElement = activator
+      // The activator should only be a valid element (Ignore comments and text nodes)
+      this.activatorElement = activator && activator.nodeType === 1 ? activator : null
```

Comment:
```suggestion
      this.activatorElement = activator?.nodeType === Node.ELEMENT_NODE
```

Reply:
Won't this just return `true` or `false`?

Reply:
Oh yeah of course :facepalm:. You got what I meant at least lol